Library/ViewModels/PledgeCTAContainerViewViewModelTests.swift

```diff
@@ -167,4 +169,19 @@ internal final class PledgeCTAContainerViewViewModelTests: TestCase {
     self.vm.inputs.configureWith(value: (.right(.couldNotParseJSON), false))
     self.pledgeRetryButtonIsHidden.assertValues([true, false])
   }
+
+  func testNotifyDelegateCTATapped() {
```

Comment:
Can we add a test to verify that tapping the button triggers the events in Koala?

Reply:
Yup, will do!